src/omero_zarr/cli.py

```diff
@@ -101,14 +108,13 @@ def _configure(self, parser):
         masks.add_argument(
             "--label-name",
             help=(
-                "Name of the array that will be stored. "
-                "Ignored for --style=split"
+                "Name of the array that will be stored. " "Ignored for --style=split"
```

Comment:
```suggestion
                "Name of the array that will be stored. Ignored for --style=split"
```

Reply:
This seems to be an interesting behavior from `black`. Thanks.